### ISSUE TITLE
Console Client crashes/disconnect when using the /inventory click command

### ISSUE THREAD

**Issue by ReinforceZwei** [ORelio/Minecraft-Console-Client#912]
I am still looking at the code and try to fix it.
related:
#903 
```
>/inventory 0 click 23

未處理的例外狀況: System.Collections.Generic.KeyNotFoundException: 指定的索引鍵
不在字典中。
   於 System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   於 MinecraftClient.Protocol.Handlers.Protocol18Handler.SendClickWindow(Int32
windowId, Int32 slotId, Item item) 於 C:\Users\reinforce\source\repos\Minecraft- Console-Client-master\MinecraftClient\Protocol\Handlers\Protocol18.cs: 行 1312
   於 MinecraftClient.McTcpClient.ClickWindowSlot(Int32 windowId, Int32 slotId)
於 C:\Users\reinforce\source\repos\Minecraft-Console-Client-master\MinecraftClie nt\McTcpClient.cs: 行 1326
   於 MinecraftClient.Commands.Inventory.Run(McTcpClient handler, String command , Dictionary`2 localVars) 於 C:\Users\reinforce\source\repos\Minecraft-Console-C lient-master\MinecraftClient\Commands\Inventory.cs: 行 46
   於 MinecraftClient.McTcpClient.PerformInternalCommand(String command, String&  response_msg, Dictionary`2 localVars) 於 C:\Users\reinforce\source\repos\Minecr aft-Console-Client-master\MinecraftClient\McTcpClient.cs: 行 372
   於 MinecraftClient.McTcpClient.CommandPrompt() 於 C:\Users\reinforce\source\r
epos\Minecraft-Console-Client-master\MinecraftClient\McTcpClient.cs: 行 275
   於 System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   於 System.Threading.ExecutionContext.RunInternal(ExecutionContext executionCo ntext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   於 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, C ontextCallback callback, Object state, Boolean preserveSyncCtx)
   於 System.Threading.ExecutionContext.Run(ExecutionContext executionContext, C ontextCallback callback, Object state)
   於 System.Threading.ThreadHelper.ThreadStart()
```

**Comment by ReinforceZwei** [comment]
Another issues.
Server kicked me when I try to click on an exist item.
```
test lost connection: Internal Exception: io.netty.handler.codec.DecoderException: net.minecraft.server.v1_15_R1.ReportedException: Loading NBT data
```

**Comment by ORelio** [comment]
The NBT encoding function is brand new in MCC, I wrote it when adding the ClickItem packet. So it is very likely that the produced NBT is actually invalid because improperly encoded 😆

**Comment by ORelio** [comment]
NBT encoding should be fixed in commit dbe02c063c12fe88930d86a98f823a6875ecbc4b:
 - Root NBT tag name was missing
 - Field type was inverted with field name
